### PR TITLE
Fix the simple-hello-world example README

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

File: simple-hello-world-example/README.md [hunk 1]
```diff
@@ -1,42 +1,32 @@
-# simple-python-webserver-blueprint
+# Simple Hello-World Example
 
-The blueprint installs a webserver on your local machine. It is supported on Linux and most *nix machines.
+This blueprint installs a web-server on the Cloudify Manager machine. It is supported on Linux and most *nix machines.
```
Comment (EarthmanT) [inline simple-hello-world-example/README.md:3]: web-server? or web server?

File: simple-hello-world-example/README.md [hunk 2]
```diff
@@ -1,42 +1,32 @@
...
+- [Cloudify CLI](https://docs.cloudify.co/latest/install_maintain/installation/installing-cli/) installed on your computer. It is used to run commands on the Cloudify Manager.
+- The Cloudify Manager machine's firewall should allow HTTP connections on port 8000.
 
 
 ## Usage
+ 
+### Install 
 
-* Clone the repository
+The `cfy install <blueprint-path>` command will automatically upload the blueprint to your Cloudify Manager, create a deployment out of it, and install the created deployment.
```
Comment (EarthmanT) [inline simple-hello-world-example/README.md:16]: automatically is probably redundant

File: simple-hello-world-example/README.md (hunk 1 above)
Comment (YoniItzhak) [reply simple-hello-world-example/README.md:3]: According to the results on google, it should be "web server", so I think "web-server" should be good.

File: simple-hello-world-example/README.md (hunk 2 above)
Comment (YoniItzhak) [reply simple-hello-world-example/README.md:16]: Right, removed.

**Review by EarthmanT — APPROVED** [review]

**Pull request merged**